requirements.txt

```diff
@@ -1,2 +1 @@
-slackeventsapi>=2.1.0
```

Comment:
Running 8158cb7:

```
    import requests
ModuleNotFoundError: No module named 'requests'
```

So looks like we need to add an explicit `requests` dependency now that we've dropped this.

Reply:
done